tornadoAPI.py

```diff
@@ -0,0 +1,441 @@
+import datetime
+import os
+import traceback
+from time import sleep
+import logging
+
+from dotenv import load_dotenv
+from selenium import webdriver
+from selenium.common.exceptions import NoSuchElementException, TimeoutException
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.wait import WebDriverWait
+
+from helperAPI import (
+    Brokerage,
+    check_if_page_loaded,
+    getDriver,
+    killSeleniumDriver,
+    printAndDiscord,
+    printHoldings,
+    stockOrder,
+)
+
+load_dotenv()
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def tornado_error(driver, loop=None):
+    driver.save_screenshot(f"Tornado-error-{datetime.datetime.now()}.png")
+    printAndDiscord(f"Tornado Error: {traceback.format_exc()}", loop, embed=False)
```

Comment:
I think `tornado_error()` should be used for every error and the logger should be removed all together for standard prints so that the code base is consistent. If you disagree, let me know.